doc/api/http.markdown

```diff
@@ -43,12 +43,6 @@ list like the following:
       'Host', 'mysite.com',
       'accepT', '*/*' ]
 
-## http.METHODS
-
-* {Array}
-
-A list of the HTTP methods that are supported by the parser.
-
```

Comment:
>I don't see how standard verb like `MKCALENDAR` is any different from non-standard one like `HELLOWORLD`.

One you need to make a calendar for and the other you should probably respond with a HELLO. If you're a CalDAV app you should respond to `MKCALENDAR` by making a calendar and respond to `HELLOWORLD` with a 405 Invalid Method. But majority of people (if not all) using this will respond to `MKCALENDAR` by NOT making a calendar and also probably not responding with 405, therefore not doing what the client that made the request expected.